src/components/RightHandMenu/LinkToSource/LinkToSource.css

```diff
@@ -1,7 +1,6 @@
 .rt__link {
   color: black;
-  font-size: 10px;
-  padding-left: 5px;
-  font-weight: 500;
+  font-size: 8px;
```

Comment:
Even though if the user changed the font size of this app it would probably break everything, I still think it's best practice to leave the font size as REM so I think `0.5rem`. Plus it's consistent with all the other font-size declarations.

Reply:
Ah, great catch! It should definitely be REM and not px, I'll make a quick change later today

Reply:
Pushed fix 👍🏻